### PR TITLE
add capability of configuring one GRE and multiple VXLAN interfaces

### PR DESCRIPTION
**Pull request by grundrauschen**

### CODE REVIEW

File: templates/deployment.yaml [hunk 1]
```diff
@@ -65,15 +65,56 @@ spec:
...
+            add: ["NET_ADMIN"]
+      {{ end }}
+
       containers:
       - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         ports:
         - containerPort: 500
+          name: u500
```
Comment (hsychla) [inline templates/deployment.yaml]: duplicate `name` line

File: templates/deployment.yaml [hunk 2]
```diff
@@ -65,15 +65,56 @@ spec:
...
           protocol: UDP
           name: u500
         - containerPort: 4500
+          name: u4500
```
Comment (hsychla) [inline templates/deployment.yaml]: duplicate `name` line

File: README.md [hunk 3]
```diff
@@ -1,2 +1,49 @@
...
+```yaml
+vxlan:
+  enabled: true
+  connectors:
+    - name: conn1
+      peer: <ip or fqdn of peer>
+      bridge: true
+      id: 42
+      ipaddr: <ip address added to the created interface>
+      bridged_ifaces: eth1 # mandatory when bridge: true
```
Comment (hsychla) [inline README.md]: the variable is plural but the value is only one interface?

File: templates/deployment.yaml [hunk 4]
```diff
@@ -142,13 +183,54 @@ spec:
...
+            add:
+              - NET_ADMIN
+        env:
+        - name: PEER
+          value: "{{ $element.peer }}"
+        {{- if $element.bridge }}
+        - name: BRIDGE
+          value: "{{ $element.bridge }}"
+        {{- end }}
+        - name: BRIDGED_IFACES
```
Comment (hsychla) [inline templates/deployment.yaml]: see readme: plural name vs. singular value

File: values.yaml [hunk 5]
```diff
@@ -50,3 +50,23 @@ resources:
   requests:
    cpu: 100m
    memory: 128Mi
+
+## Values used to set up GRE usage
+gre:
+  enabled: false
+  localip: 192.0.2.1
+  remoteip: 203.0.113.0
+  name: gre0
```
Comment (hsychla) [inline values.yaml]: The readme recommends not to use `gre0` but you use it as default...

File: templates/deployment.yaml (hunk 4 above)
Comment (grundrauschen) [reply templates/deployment.yaml]: multiple values are supported in a space separted list, eg. "eth1 eth2 net0". I clearified this in the README.

**Pull request merged**